week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,17 +5,26 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(Arr) {
+let String=Arr.map(tidy=>tidy.trim().toLowerCase().replace("/",""));
 
+return String; 
+}
 /*
 Complete the function to check if the variable `num` satisfies the following requirements:
 - is a number
-- is even
+- is even 
 - is less than or equal to 100
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+if (typeof num === "number" && num % 2 === 0 && num <= 100) {
+  return true
+}else{ 
+  return false
+}
+} 
```

Comment:
Please indent your code.

It's a correct solution but do we really need the if-else statement?
Think about it, what is the outcome type of that logical expression there? In other words, what type of value will it result?